src/components/interaction-designer/ErrorNotifications.vue

```diff
@@ -0,0 +1,119 @@
+<!--      </button>-->
+<!--    </section>-->
+    <section class="alert alert-danger d-flex notification-panel" role="alert" v-if="numberOfBlocksWithErrors > 0">
+      <span class="align-self-center ml-2">
+        You have {{numberOfBlocksWithErrors}} block(s) that has validation issue.
+      </span>
+      <div class="dropdown">
+        <button class="btn btn-link dropdown-toggle" type="button" id="dropdownMenu1" data-toggle="dropdown"
+                aria-haspopup="true" aria-expanded="true">
+          Locate blocks to fix issue
+          <span class="caret"></span>
+        </button>
+        <ul class="notification dropdown-menu" aria-labelledby="dropdownMenu1">
+          <li v-for="(status, key) in blockValidationStatuses">
+            <div class="card" >
+              <div class="card-title m-0 p-2">
+                {{ (status.type).split('.')[1] }}
+              </div>
+              <div class="card-body d-flex justify-content-between" v-for="error in status.ajvErrors">
+                <span class="text-danger align-self-center">{{ error.dataPath }} - {{ error.message }}</span>
+                <button type="button" class="btn btn-link" @click="fixBlockError(key, error.dataPath)">
+                  Fix Issue
+                </button>
+              </div>
+            </div>
+          </li>
+        </ul>
+      </div>
+    </section>
+  </main>
+</template>
+
+<script lang="ts">
+import Lang from '@/lib/filters/lang';
+import { pickBy, values } from 'lodash';
+import { IValidationStatus } from '@/store/validation';
+import Routes from '@/lib/mixins/Routes';
+import Component, { mixins } from 'vue-class-component';
+import { namespace } from 'vuex-class';
+
+const validationVuexNamespace = namespace('validation')
+
+@Component({})
+export default class ErrorNotifications extends mixins(Routes, Lang) {
+  get flowValidationErrors() {
+    const flowValidationResults: { [key:string]: IValidationStatus } = pickBy(this.validationStatuses, function(value: IValidationStatus) {
+      return value.type === 'flow'
+    })
+    return values(flowValidationResults)[0]?.ajvErrors || []
+  }
+
+  get blockValidationStatuses() {
+    return pickBy(this.validationStatuses, function (value: IValidationStatus) {
+      return value.type !== 'flow'
+    })
+  }
+
+  get numberOfBlocksWithErrors() {
+    return values(this.blockValidationStatuses).length
+  }
+
+  fixFlowError() {
+    this.$router.push({
+      name: 'flow-details',
+    })
+  }
+
+  fixBlockError(key: string, dataPath: string) {
+    const blockId = key.replace('block/', '');
+    this.$router.push({
+      name: 'block-scroll-to-anchor',
+      params: {
+        blockId,
+        field: dataPath
+      },
+    })
+  }
+
+  @validationVuexNamespace.Getter validationStatuses!: { [key: string]: IValidationStatus }
+}
+</script>
+
+<style scoped lang="scss">
+.error-notifications-wrapper {
+  position: fixed;
+  width: 100vw;
+  top: 60px;
+  z-index: 2 * 10;
```

Comment:
and to fix hidden menu too
<img width="309" alt="Screen Shot 2021-05-27 at 2 14 44 PM" src="https://user-images.githubusercontent.com/68745918/119891606-69ba9580-bef6-11eb-8595-a4e6430ab6f9.png">

Reply:
That is interesting!  In my machine it works differently.  Just to confirm, did you test this on the latest code?

![Screen Shot 2021-05-27 at 4 37 28 PM](https://user-images.githubusercontent.com/23651142/119911189-a5148e80-bf0d-11eb-8b62-352bc13f2e19.png)

Reply:
Also, personally I would prefer the block to be hidden under the error banner than it being on top of it. My reason for it is that the user can scroll to see the block but if it overlaps on the error link it would be harder to see the errors.  Let me know your thoughts.